src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -379,7 +393,7 @@ static NamedArgsAndClosure parseArgs(Object arg, StepDescriptor d) {
      *      If the context in which this method call happens allow implicit sole default argument, specify its name.
      *      If null, the call must be with names arguments.
      */
-    static NamedArgsAndClosure parseArgs(Object arg, boolean expectsBlock, String soleArgumentKey) {
+    static NamedArgsAndClosure parseArgs(Object arg, boolean expectsBlock, String soleArgumentKey, boolean singleRequiredArg) {
```

Comment:
Are you sure there is no usages in other core Pipeline plugins. IIRC there were overlaps